commit 9f7df2a39ab8aad4eacc4e35e80d94f3b273637f
Author: Adam Ford <aford173@gmail.com>
Date:   Sun Oct 28 15:29:27 2018 -0500

    ARM: dts: logicpd-somlv: Fix interrupt on mmc3_dat1
    
    [ Upstream commit 3d8b804bc528d3720ec0c39c212af92dafaf6e84 ]
    
    The interrupt on mmc3_dat1 is wrong which prevents this from
    appearing in /proc/interrupts.
    
    Fixes: ab8dd3aed011 ("ARM: DTS: Add minimal Support for Logic PD
    DM3730 SOM-LV") #Kernel 4.9+
    
    Signed-off-by: Adam Ford <aford173@gmail.com>
    Signed-off-by: Tony Lindgren <tony@atomide.com>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/arch/arm/boot/dts/logicpd-som-lv.dtsi b/arch/arm/boot/dts/logicpd-som-lv.dtsi
index ac343330d0c8..98b682a8080c 100644
--- a/arch/arm/boot/dts/logicpd-som-lv.dtsi
+++ b/arch/arm/boot/dts/logicpd-som-lv.dtsi
@@ -129,7 +129,7 @@
 };
 
 &mmc3 {
-	interrupts-extended = <&intc 94 &omap3_pmx_core2 0x46>;
+	interrupts-extended = <&intc 94 &omap3_pmx_core 0x136>;
 	pinctrl-0 = <&mmc3_pins &wl127x_gpio>;
 	pinctrl-names = "default";
 	vmmc-supply = <&wl12xx_vmmc>;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1028_9f7df2a39ab8aad4eacc4e35e80d94f3b273637f.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1028_9f7df2a39ab8aad4eacc4e35e80d94f3b273637f.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
